### PR TITLE
version 3.1

### PR DESCRIPTION
**Pull request by ultrabug**
plz review changelog

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,28 @@
+version 3.1 (2016-09-09)
+* NOTE: the documentation is getting stronger, check it out to know more about the new features listed
+* @tobes has become the first collaborator of the py3status project !
+* new format parser allows for rich and conditional format strings, by @tobes
+* it is now possible to use obfuscation on all module parameters, by @tobes
```
Comment (tobes) [inline CHANGELOG]: all __string__ module parameters

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+* fix: tztime modules glitch
+* fix: some crashes due to unicode logging
+* new module testing framework allowing feature rich standalone tests of modules, by @tobes
+* new tcp_status module to check for a given TCP port on a host, by @ritze
+* new file_status module to check the presence of a file, by @ritze
+* new process_status module to check the presence of a process, by @ritze
+* new kdeconnector module to display information about Android devices, by @ritze
+* new mpris module to control MPRIS enabled music players, by @ritze
+* new frame module to group and display multiple modules inline, by @tobes
+* new rainbow module to enlight your day, by @tobes
```
Comment (tobes) [inline CHANGELOG]: :)

**Comment by tobes** [comment]
Cool, I'll try to see if you missed anything over the weekend but looking good

**Comment by tobes** [comment]
I think it would be good to mention these contributions to the project

Fix for when config-file happened to be a symbolic link by @Stautob
Improved detection of active battery in battery_level module by @ZeiP
volume_status now shows as color bad when muted by @chezstov
New  module gpmdp for Google Play Music Desktop Player module by @Spirotot
process_status module functionality improvements by @ykhodorkovskiy

Also

i3status.conf file is read with a new parser which allows for richer objects eg lists to be used in config, better unicode support for python2 and better reporting to the user for errors in the config.

**Comment by ultrabug** [comment]
done !

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,35 @@
+version 3.1 (2016-09-09)
+* NOTE: the documentation is getting stronger, check it out to know more about the new features listed
+* @tobes has become the first collaborator of the py3status project !
+* i3status.conf file is read with a new parser which allows for richer objects eg lists to be used in config, better unicode support for python2 and better reporting to the user for errors in the config, by @tobes
```
Comment (tobes) [inline CHANGELOG]: Can we split this line?

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,35 @@
+version 3.1 (2016-09-09)
```
Comment (tobes) [inline CHANGELOG]: Do you want to update this too.

**Comment by ultrabug** [comment]
About the line split, I think it's okay to leave it on a changelog

**Comment by tobes** [comment]
Ok, well I think we are all good on this release then :tada:

**Pull request merged**